mkdocs.yml

```diff
@@ -6,7 +6,8 @@ docs_dir: docs/
 repo_name: GitHub
 repo_url: https://github.com/k0sproject/k0s
 copyright:
-  Copyright &copy; 2021 <a href="https://mirantis.com/">Mirantis Inc.</a>
+  Copyright &copy; 2021 k0sproject a Series of LF Projects, LLC.
+  For website terms of use, trademark policy and other project policies please see lfprojects.org/policies/.
```

Comment:
I wonder if we should make this a proper link.

Reply:
I tried. mkdocs doesn't render markdown on the copyright 😢